### DISCUSSION TITLE
How to use RaisePropertyChanged for StyledProperty?

### DISCUSSION THREAD

**Discussion by SwiftExtender** [AvaloniaUI/Avalonia#15813] [Q&A] [answered]
How to use RaisePropertyChanged for StyledProperty? 

Af far as i see, first parameter of RaisePropertyChanged  must be DirectPropertyBase. How can i pass StyledProperty as first argument?

**Comment by workgroupengineering** [comment, marked as answer]
it is automatically handled by `SetValue` or `SetCurrentValue`

**Reply by robloo** [reply]
Note that there are edge cases where things don't work https://github.com/AvaloniaUI/Avalonia/issues/10092. We don't have a public API to force a property changed notification in these cases. There is one used internally.

**Comment by SwiftExtender** [comment]
@workgroupengineering @robloo thanks
